scripts/adb_devices_abis.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
And please use `env` in shebangs:
```suggestion
#!/usr/bin/env bash
```